src/raster/r.forcircular/r.forcircular

```diff
@@ -0,0 +1,2039 @@
+#!/usr/bin/env python3
+# -- coding: utf-8 --
+#
+############################################################################
+#
+# MODULE:      r.forcircular
+# AUTHOR(S):   Francesco Geri, Sandro Sacchelli
+# PURPOSE:     Evaluation of circular bioconomy level of forest ecosystems
+# COPYRIGHT:   (C) 2022 by the GRASS Development Team
+#
+#              This program is free software under the GNU General Public
+#              License (>=v2). Read the file COPYING that comes with GRASS
+#              for details.
+#
+#############################################################################
+#%Module
+#% description: Evaluation of circular bioconomy level of forest ecosystems
+#% keyword: bioeconomy
+#% keyword: forest
+#% overwrite: yes
```

Comment:
```suggestion
#% overwrite: yes
```

AFAIK this isn't needed

Reply:
Sorry Markus, I'm not sure to understand: I have to remove all rows with "#% overwrite: yes"?

Reply:
It is a single row to be deleted unless there is a reason to keep it.

Reply:
Ok. Thank you very much.

Reply:
Done